lib/user_agent/browsers/acast.rb

```diff
@@ -0,0 +1,63 @@
+class UserAgent
+  module Browsers
+    # Acast/1.63.0 (Phone; iOS 11.2.6; iPad6,11)
+    # Acast/1.63.0 (Phone; iOS 11.2.5; iPhone9,4)
+    # Acast/1.42.1 (phone; android 10; Google Pixel 3 XL)
+    # Acast/1.26.0 (Linux;Android 9) ExoPlayerLib/2.2.0
+    class Acast < Base
+      ACAST_REGEX   = /Acast/
+      ANDROID_REGEX = /[Aa]ndroid/
+      IOS_REGEX     = /iOS/
+      IPAD_REGEX    = /iPad/
+
+      def self.extend?(agent)
+        agent.detect { |useragent| ACAST_REGEX.match?(useragent.product) }
+      end
+
+      def browser
+        'Acast'
+      end
+
+      # Gets the right application
+      def application
+        detect_product('Acast')
+      end
+
+      ##
+      # @return [Boolean] true for mobile
+      def mobile?
+        true
+      end
+
+      # Gets the operating system
+      #
+      # @return [String] the os
```

Comment:
```suggestion
      # @return [String, nil] the os
```
Same for the other browsers